examples/Example/src/new_api/overlap_parent/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import {
+  GestureMonitor,
+  useGesture,
+  Gesture,
+} from 'react-native-gesture-handler';
+
+function Box(props: {
+  color: string;
+  overlap?: boolean;
+  children?: React.ReactNode;
+}) {
+  const gesture = useGesture(
+    Gesture.tap().setOnEnd((_e, success) => {
+      if (success) console.log(props.color);
```

Comment:
```suggestion
      if (success) { console.log(props.color) };
```

Reply:
Done in 1e19778e795f293d5f933ab6dfa528b78373a64f